m2py/utils/post_processing.py

```diff
@@ -126,7 +130,9 @@ def show_classification_distributions(labels, data, data_type, title_flag=True):
         ax_r.set_title("Distributions")
         for index, j in enumerate(unique_labels):
             color_step = num_labels - (index + 1)
-            ax_r.hist(data[:, :, i][labels == j], NUM_BINS, alpha=ALPHA, density=True, color=cmap(color_step))
+            labels_wo_outliers = labels * (1 - outliers)
+            class_data = data[:, :, i][labels_wo_outliers == j]
```

Comment:
This is fixing a bug. This excludes the outliers from the distribution plots. This should remove high peaks in the distributions going forward.